### PR TITLE
MAR-4611: Upgrade to Python 3

### PR DESCRIPTION
**Pull request by Reshma-M-A**

### CODE REVIEW

**Comment by Reshma-M-A** [comment]
So far looks good to me.

**Review by tobiasbrown — APPROVED** [review]

File: tests/app/views/test_marketplace.py [hunk 1]
```diff
@@ -267,7 +267,7 @@ def test_buyer_start_application(self):
         brief_id = self.brief['briefs']['id']
         res = self.client.get(self.expand_path('/digital-service-professionals/opportunities/{}'.format(brief_id)))
         assert_equal(200, res.status_code)
-        assert_not_equals(res.data.title().strip().find("/Sellers/Opportunities/1/Ask-A-Question"), -1)
+        assert_not_equals(res.data.title().strip().find("/Sellers/Opportunities/1/Ask-A-Question".encode()), -1)
```
[the remark this answers is not in the document]
Comment (Revit) [reply tests/app/views/test_marketplace.py:271]: It is a test file it is ok

File: tests/app/views/test_marketplace.py [hunk 2]
```diff
@@ -280,7 +280,7 @@ def test_supplier_start_application(self):
         res = self.client.get(self.expand_path('/digital-service-professionals/opportunities/{}'.format(brief_id)))
         assert_equal(200, res.status_code)
 
-        assert_not_equals(res.data.title().strip().find("/Sellers/Opportunities/1/Ask-A-Question"), -1)
+        assert_not_equals(res.data.title().strip().find("/Sellers/Opportunities/1/Ask-A-Question".encode()), -1)
```
[the remark this answers is not in the document]
Comment (Revit) [reply tests/app/views/test_marketplace.py:284]: It is a test file it is ok

**Review by Revit — APPROVED** [review]

**Comment by Reshma-M-A** [comment]
code looks good

**Pull request merged**